.github/gh_matrix_builder.py

```diff
@@ -156,34 +156,15 @@ def macos_config(overrides):
     return base_config
 
 
-# common ignored tests for all scheduled pg15 tests
-ignored_tests = {}
-
-# common ignored tests for all non-scheduled pg15 tests (e.g. PRs)
-if pull_request:
-    ignored_tests = {
-        "telemetry",
```

Comment:
Was this intentionally removed?

Reply:
Yes, it was intentionally removed because it was leading to broken IGNORES during the CI execution. So I've rearranged the code to accumulate the ignores instead of override them.

Reply:
I actually meant the telemetry test removal itself. Was that intentional? Should it be kept in the IGNORES?